django_simple_bulma/extensions/bulma-dropdown/dist/js/bulma-dropdown.js

```diff
@@ -0,0 +1,40 @@
+(function() {
+    window.dropdowns = {};
+
+    function handle_event(e) {
+        show();
+
+        function clickEvent() {
+            hide();
+            document.body.removeEventListener("click", clickEvent);
+        };
+
+        document.body.addEventListener("click", clickEvent);
+
+        e.stopPropagation();
+    }
+
+    let elements = document.getElementsByClassName("dropdown");
+
+    for (let element of elements) {
+
+        // We shouldn't handle hoverable menues
+        if (element.classList.contains("is-hoverable")) {
+            continue;
+        }
+
+        let menu_element = element.getElementsByClassName("dropdown-menu")[0];
+
+        function show() {
+            element.classList.add("is-active");
+        }
+
+        function hide() {
+            element.classList.remove("is-active");
+        }
+
+        element.addEventListener("click", handle_event);
+
+        window.dropdowns[menu_element.id] = element;
```

Comment:
I'm unsure if these `window.dropdowns` declarations are necessary or if they were just necessary specifically on pythondiscord.com. maybe @gdude2002 can shed some light on that?

Reply:
It shouldn't be necessary

Reply:
I left it in because it is neat for debugging

Reply:
If we're going to leave it in, we should probably at least document it so that our users will know that it's "neat for debugging". I don't have a very strong opinion on whether to remove it or document it, I'll leave it for you to decide.

Reply:
I got rid of it